source/retryable-reads/tests/unified/handshakeError.yml

```
@@ -126,4 +127,681 @@ tests:
               command:
                 find: *collectionName
                 filter: { _id: 2 }
+              commandName: find
               databaseName: *databaseName
+
+  - description: "aggregate succeeds after retryable handshake network error"
+    operations:
+      - name: failPoint # fail the next connection establishment
+        object: testRunner
+        arguments:
+          client: *client0
+          failPoint:
+            configureFailPoint: failCommand
+            mode: { times: 2 }
+            data:
+              failCommands: [saslContinue, ping]
+              closeConnection: true
+
+      - name: runCommand
+        object: *database0
+        arguments:
+          commandName: ping
+          command: { ping: 1 }
+        expectError:
+          isError: true
+
+      - name: aggregate
+        object: *collection0
+        arguments:
+          pipeline: [{ $match: { _id: { $gt: 1 } } }]
+          batchSize: 2
+        expectResult:
+          - { _id: 2, x: 22 }
+          - { _id: 3, x: 33 }
+
+    expectEvents:
+      - client: *client0
+        eventType: cmap
+        events:
+          - { connectionCheckOutStartedEvent: {} }
+          - { connectionCheckOutStartedEvent: {} }
+          - { connectionCheckOutStartedEvent: {} }
+          - { connectionCheckOutStartedEvent: {} }
+      - client: *client0
+        events:
+          - commandStartedEvent:
+              command:
+                ping: 1
+              databaseName: *databaseName
+          - commandStartedEvent:
+              command:
+                aggregate: *collectionName
+                pipeline: [{ $match: { _id: { $gt: 1 } } }]
+                cursor: { batchSize: 2 }
+              commandName: aggregate
+              databaseName: *databaseName
+          - commandStartedEvent:
+              command:
+                getMore: { $$type: [int, long] }
+                collection: *collectionName
+                batchSize: 2
+              commandName: getMore
+              databaseName: *databaseName
+
+  - description: "aggregate succeeds after retryable handshake network error (ShutdownInProgress)"
```

```suggestion
  - description: "aggregate succeeds after retryable handshake server error (ShutdownInProgress)"
```

The `aggregate succeeds after retryable handshake network error` case tests an error due to the server closing the connection. This case tests an error reply from the server. Suggest updating all `ShutdownInProgress` cases.